README.md

```diff
@@ -101,8 +101,8 @@ To build Jupyter Lite with this kernel without creating a website you can execut
 ```bash
 micromamba create -n xeus-lite-host jupyterlite-core
 micromamba activate xeus-lite-host
-python -m pip install jupyterlite-xeus
-jupyter lite build --XeusAddon.prefix=$PREFIX
+python -m pip install jupyterlite-xeus jupyter_server
```

Comment:
Why add jupyter_server here ? Does it solve a warning or something ?

Reply:
Ahh actually I got it. This is due to https://github.com/jupyterlite/jupyterlite/issues/1402

Reply:
Wait a better way to get jupyter_server is through micromamba 

use 
`micromamba create -n xeus-lite-host jupyterlite-core jupyter-server` and then let's move this in